main.go

```diff
@@ -75,6 +114,37 @@ func tcpEventCb(event tcpEvent) {
 	lastTimestamp = timestamp
 }
 
+func tcpEventCbV6(event tcpEventV6) {
+	timestamp := uint64(event.Timestamp)
+	cpu := event.Cpu
+	typ := event.Type
+	pid := event.Pid & 0xffffffff
+
+	saddrbuf := make([]byte, 16)
+	daddrbuf := make([]byte, 16)
+
+	binary.LittleEndian.PutUint64(saddrbuf, event.SAddrH)
+	binary.LittleEndian.PutUint64(saddrbuf[4:], event.SAddrL)
+	binary.LittleEndian.PutUint64(daddrbuf, event.DAddrH)
+	binary.LittleEndian.PutUint64(daddrbuf[4:], event.DAddrL)
+
+	sIP := net.IP(saddrbuf)
+	dIP := net.IP(daddrbuf)
+
+	sport := event.SPort
+	dport := event.DPort
+	netns := event.NetNS
+
+	fmt.Printf("%v cpu#%d %s %v %v:%v %v:%v %v\n", timestamp, cpu, typ, pid, sIP, sport, dIP, dport, netns)
+
+	if lastTimestamp > timestamp {
+		fmt.Printf("WARNING: late event!\n")
+		os.Exit(1)
+	}
+
+	lastTimestamp = timestamp
```

Comment:
If you want to keep the warning, you need one lastTimestamp for v4 and one for v6 because they come from different maps. The reordering I implemented in only between perf fds within a map, not global on all maps.

For Scope, I don't think we need ordering between v4 events and v6 events. If we do, then we need to use only one perf map.